docs/_generate/index.html

```diff
@@ -488,6 +489,67 @@ <h3 class='space-bottom1'><a href='#sources-video' title='link to video'>video</
             </tbody>
           </table>
         </div>
+        <div id='sources-canvas' class='pad2 keyline-bottom'>
+          <h3 class='space-bottom1'><a href='#sources-canvas' title='link to canvas'>canvas</a></h3>
+          <p>
+            A canvas source. The <code>"canvas"</code> value is the ID of the canvas element in the document.
```

Comment:
Ah, I guess this is an inconsistency between the source and layer documentation. Fodder for other PRs, I guess.